docs/Lexicon.md

```diff
@@ -594,6 +594,17 @@ of `Optional<Int>`.) Sugared types preserve information about the form
 and use of the type even though the behavior usually does not change
 (except for things like access control). Contrast with [canonical type](#canonical-type).
 
+## TBD
+
+Text-based dynamic library files (TBDs) are a textual representation of
+the information in a dynamic library / shared library that is required
+by the static linker - basically a symbol list of the exported symbols.
+
+Apple’s SDKs originally used Mach-O Dynamic Library Stubs. Mach-O Dynamic
+Library Stubs are dynamic library files, but with all the text and data
+stripped out. TBD files were introduced to replaced Mach-O Dynamic Library
+Stub files in the SDK to further reduce its overall size.
```

Comment:
I would drop the last sentence at least - unless you have the ability to cite this as the case.